src/background/utils/overwriteWitnetNodeConfiguration.ts

```diff
@@ -1,30 +1,60 @@
 import path from 'path'
 import fs from 'fs-extra'
 
+const nodeUrlUntilCharacter = (character: string) =>
+  new RegExp('node_url =([^;]*)' + character)
+
+const nodeUrlsRegex = /g\[([^;]*)\]|\"([^;]*)\"/
+
+export function hasOldWitnetIp({
+  oldWitnetNodes,
+  currentNodeConfiguration,
+}: {
+  oldWitnetNodes: Array<string>
+  currentNodeConfiguration: Array<string>
+}) {
+  return currentNodeConfiguration.some(ip => oldWitnetNodes.includes(ip))
+}
+
 // Replace witnet nodes urls in witnet configuration file
 export function overwriteWitnetNodeConfiguration({
   sheikahPath,
   witnetConfigFileName,
   publicNodeUrls,
+  oldWitnetNodes,
 }: {
   sheikahPath: string
   witnetConfigFileName: string
   publicNodeUrls: Array<string>
+  oldWitnetNodes: Array<string>
 }) {
```

Comment:
We are passing always the same arguments to this function. I like being able to pass them to be able to test the function easily, but could we add a default argument value?

```suggestion
const defaultNodeConfiguration = {
    sheikahPath: SHEIKAH_PATH,
    witnetConfigFileName: WITNET_CONFIG_FILE_NAME,
    publicNodeUrls: URLS_PUBLIC_WITNET_NODES,
    oldWitnetNodes: OLD_WITNET_NODE_IPS,
}

export function overwriteWitnetNodeConfiguration({
  sheikahPath,
  witnetConfigFileName,
  publicNodeUrls,
  oldWitnetNodes,
}: {
  sheikahPath: string
  witnetConfigFileName: string
  publicNodeUrls: Array<string>
  oldWitnetNodes: Array<string>
} = defaultNodeConfiguration) {
```